Framework/NexusGeometry/inc/MantidNexusGeometry/TubeHelpers.h

```diff
@@ -29,7 +29,7 @@ findAndSortTubes(const Mantid::Geometry::IObject &detShape,
                  const std::vector<Mantid::detid_t> &detIDs);
 MANTID_NEXUSGEOMETRY_DLL std::vector<Mantid::detid_t>
 notInTubes(const std::vector<detail::TubeBuilder> &tubes,
-           const std::vector<Mantid::detid_t> &detIDs);
+           std::vector<Mantid::detid_t> detIDs);
```

Comment:
Hi David, not sure what you mean here. Do you want me to change something? I just changed it to pass the detIDs parameter by value so that I could sort the vector inside notInTubes. I could have just removed the "const" and left it as passing by reference but sorting the supplied vector seemed like an unexpected side effect the caller may not want

Reply:
:man_facepalming: Ah I didn't see the sorting step, so assumed we were making a copy by accident.